radix-transaction-scenarios/src/scenarios/account_locker.rs

```diff
@@ -1,17 +1,15 @@
 use crate::internal_prelude::*;
-use radix_engine_interface::{
-    blueprints::account::{
-        AccountSetResourcePreferenceInput, ResourcePreference,
-        ACCOUNT_SET_RESOURCE_PREFERENCE_IDENT,
-    },
-    *,
-};
+use crate::utils::*;
+use radix_engine::updates::ProtocolUpdate;
+use radix_engine_interface::blueprints::account::*;
+use radix_engine_interface::*;
 
 pub struct AccountLockerScenarioConfig {
-    pub account_locker_admin_account: VirtualAccount,
-    pub user_account1: VirtualAccount,
-    pub user_account2: VirtualAccount,
-    pub user_account3: VirtualAccount,
+    pub account_locker_admin_account: PrivateKey,
```

Comment:
Very minor, but I think it would be a tiny bit clearer if we added the `_key` suffix to all of these